Vizzuality/trase
# Fix/clear expanded nodes

## Asana

https://app.asana.com/0/1201503005195814/1201791853439789/f

## Description

Clear expanded nodes when we click on the Clear nodes button

## Testing instructions

Select a node and expand it. Click on the clear button. The sankey should collapse
